shared/src/main/scala/minitest/runner/Runner.scala

```diff
@@ -20,24 +20,25 @@ package minitest.runner
 import sbt.testing.{Runner => BaseRunner, Task => BaseTask, _}
 
 final class Runner(
-  val args: Array[String],
-  val remoteArgs: Array[String],
+  // error overiding Array[T | JavaNull] | JavaNull with Array[T]
+  val args: Array[String | Null] | Null,
```

Comment:
**Override java method**
A Scala method overrides some Java method. We need to add |Null to the method's parameter types to make the override still work.

Number of occurrences in file: 3